src/php/phel.php

```diff
@@ -31,13 +31,9 @@ function getHelpText(): string
     exit;
 }
 
-switch ($argv[1]) {
-    case "repl":
-        $repl = new Repl($currentDir);
-        $repl->run();
-        break;
-
-    default:
-        echo getHelpText();
-        exit;
+if ('repl' === $argv[1]) {
```

Comment:
Could we leave this as switch? Currently there is just the `repl` command but there will be more commands in the future.

Reply:
Natürlich :)